e2e/discoverSheetFlow.spec.js

```diff
@@ -0,0 +1,199 @@
+    await Helpers.clearField('import-sheet-input');
+    await Helpers.typeText('import-sheet-input', process.env.TEST_SEEDS, false);
+    await Helpers.delay(2000);
+    await Helpers.checkIfElementHasString(
+      'import-sheet-button-label',
+      'Import'
+    );
+    await Helpers.tap('import-sheet-button');
+    await Helpers.delay(2000);
+    await Helpers.checkIfVisible('wallet-info-modal');
+  });
+
+  it('Should navigate to the Wallet screen after tapping on "Import Wallet"', async () => {
+    await Helpers.delay(5000);
+    await Helpers.tap('wallet-info-submit-button');
+    if (device.getPlatform() === 'android') {
+      await Helpers.checkIfVisible('pin-authentication-screen');
+      // Set the pin
+      await Helpers.authenticatePin('1234');
+      // Confirm it
+      await Helpers.authenticatePin('1234');
+    }
+    await Helpers.delay(1000);
+    await Helpers.checkIfVisible('wallet-screen');
+  });
+
+  it('Should navigate to Discover screen after tapping Discover Button', async () => {
+    await Helpers.delay(2000);
+    await Helpers.tap('discover-button');
+    await Helpers.delay(3000);
+    await Helpers.checkIfVisible('top-movers-section');
+    await Helpers.checkIfVisible('dpi-button');
+    await Helpers.checkIfVisible('lists-section');
+    await Helpers.checkIfVisible('pools-section');
+  });
+
+  it('Should show the camera if Discover is minimized', async () => {
+    await Helpers.delay(1000);
+    await Helpers.swipe('discover-header', 'down');
+    await Helpers.delay(1000);
+    await Helpers.checkIfVisible('scanner-header');
+    await Helpers.checkIfNotVisible('pools-section');
+  });
+
+  it('Should open Discover Search on pressing search fab', async () => {
+    await Helpers.delay(1000);
+    await Helpers.tap('search-fab');
+    await Helpers.swipe('discover-header', 'up');
+    await Helpers.delay(2000);
+    await Helpers.checkIfVisible('done-button');
+  });
+
+  it('Should search and open expanded state for SOCKS', async () => {
+    await Helpers.delay(1000);
+    await Helpers.typeText('discover-search-input', 'SOCKS\n', true);
+    await Helpers.delay(8000);
+    await Helpers.checkIfVisible(
+      'discover-currency-select-list-exchange-coin-row-SOCKS'
+    );
+    await Helpers.checkIfNotVisible(
+      'discover-currency-select-list-exchange-coin-row-ETH'
+    );
+    await Helpers.tap('discover-currency-select-list-exchange-coin-row-SOCKS');
+    await Helpers.delay(2500);
+    await Helpers.checkIfVisible('chart-header-Unisocks');
+  });
+
+  it('Should add Unisocks to Watchlist & remove from Favorites', async () => {
+    await Helpers.tap('add-to-list-button');
+    await Helpers.delay(2000);
+    await Helpers.checkIfVisible('add-token-sheet');
+    await Helpers.tap('add-to-watchlist');
+    await Helpers.delay(1000);
+    await Helpers.tap('remove-from-favorites');
+    await Helpers.delay(1000);
+    await Helpers.tap('close-action-button');
```

Comment:
can you add two more actions to verify it's no longer showing under favorites, and it's showing correctly under watchlist?